src/event.ts

```diff
@@ -31,7 +31,7 @@ interface EventBase<T extends PackageReference = ScriptReference> {
 // Success events
 // -------------------------------
 
-type Success = ExitZero | NoCommand | Fresh | Cached;
+export type Success = ExitZero | NoCommand | Fresh | Cached;
```

Comment:
Nit, when exporting should add a jsdoc comment.

Reply:
Done!